src/db/UniqueTags.cxx

```diff
@@ -32,9 +32,8 @@ CollectTags(TagSet &set, TagType tag_type, TagMask group_mask,
 	    const LightSong &song)
 {
 	assert(song.tag != nullptr);
-	const Tag &tag = *song.tag;
 
-	set.InsertUnique(tag, tag_type, group_mask);
+	set.InsertUnique(song, tag_type, group_mask);
```

Comment:
What does this fix?

Reply:
Just for caryaudio.  Insert a album tag by guss. eg:
There are two songs below. They didn't have album tag.
/album_name/song_name1.wav
/song_name2.wav

origin mpd return no album tag.

caryaudio mpd return album tag below
album: album_name
album: Folder